src/components/common/ColonyActions/ColonyActions.tsx

```diff
@@ -223,6 +223,23 @@ const ColonyActions = (/* { ethDomainId }: Props */) => {
         )}
         text="Test Create Domain"
       />
+      <ActionButton
+        submit={ActionTypes.ACTION_DOMAIN_EDIT}
+        error={ActionTypes.ACTION_DOMAIN_EDIT_ERROR}
+        success={ActionTypes.ACTION_DOMAIN_EDIT_SUCCESS}
```

Comment:
Right, now I remember. This was refactored in @willm30 's #261 which has been since merged.

So this can just become:
```tsx
      <ActionButton
        actionType={ActionTypes.ACTION_DOMAIN_EDIT}
        ...
      />
```